mobile/src/components/common/checkbox/checkbox.tsx

```diff
@@ -16,12 +16,14 @@ type Props<T extends FormControlValues> = {
   name: FormControlPath<T>;
   control: FormControl<T>;
   onToggle: () => void;
+  isChecked?: boolean;
 };
 
 const Checkbox = <T extends FormControlValues>({
   name,
   control,
   onToggle,
+  isChecked,
```

Comment:
we should control this via useForm, if you need to set value after data fetching, do it via useEffect and reset